bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckablePullRequest.java

```diff
@@ -178,6 +178,11 @@ Hash commit(Hash finalHead, Namespace namespace, String censusDomain, String spo
             committer = author;
         }
 
+        var authorSet = Authors.author(pr.repository().forge().currentUser(), pr.comments());
```

Comment:
Calling `pr.comments()` here is a remote call so rather expensive. It's already done in `commitMessage`. We could at least make sure we only call it once. Looking a bit further, I see that in all places where a `CheckablePullRequest` is created, we already have a list of comments, so I think an even better solution here is to add a `comments` field and supply it to the constructor. 

I think the variable name can be clearer.
```suggestion
        var overridingAuthor = Authors.author(pr.repository().forge().currentUser(), pr.comments());
```

Reply:
Sure, will reuse the comments

Reply:
Was there a reason you didn't add the comments as a field in `CheckablePullRequest`? I think that would make more sense as the `PullRequest` already is a field.

Reply:
No. I misunderstood what you mean in the morning. I will fix it soon.